types/google-map-react/index.d.ts

```diff
@@ -99,14 +99,17 @@ declare namespace googleMapReact {
         x: number;
         y: number;
     }
-
-    interface NESWBounds {
```

Comment:
All valid `NESWBounds` are valid `NeSwDiagonal` (ignoring excess property checking), so it seems like we have a good opportunity to make this change less breaky by leaving `NESWBounds` in and marking it `/** @deprecated */`.